src/resolvers/collection.js

```diff
@@ -79,6 +79,18 @@ export default {
 
       return dataSources.graphDbSource(source, args, context, parseResolveInfo(info))
     },
+    generateVariableDrafts: async (source, args, context) => {
+      const { conceptId } = source
+
+      const { dataSources } = context
+
+      const { generateCollectionVariablesApi } = dataSources
+
+      const variableDrafts = await generateCollectionVariablesApi
```

Comment:
No need to `await` then return here, you can simply `return generateCollectionVariablesApi.generateVariables(conceptId)`, when you return an async operation the await is assumed and handled. In fact, if you were to type `return await ...` it would provide the docs on it if you wanted more info.